config/delta-producer/services.json

```diff
@@ -0,0 +1,46 @@
+{
+    "besluiten": {
+        "deltaInterval": 10000,
+        "deltaPath": "/besluiten/delta",
+        "errorCreatorUri": "http://lblod.data.gift/services/delta-producer-publication-graph-maintainer-besluiten",
+        "errorGraph": "http://mu.semte.ch/graphs/harvesting",
+        "exportConfigPath": "/config/besluiten/export.json",
+        "filesGraph": "http://mu.semte.ch/graphs/harvesting",
+        "filesPath": "/besluiten/files",
+        "healingJobOperation": "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/healingOperation/besluiten",
+        "initialPublicationGraphSyncJobOperation": "http://redpencil.data.gift/id/jobs/concept/JobOperation/deltas/initialPublicationGraphSyncing/besluiten",
+        "jobsGraph": "http://mu.semte.ch/graphs/harvesting",
+        "loginPath": "/besluiten/login",
+        "prettyPrintDiffJson": "true",
+        "publicationGraph": "http://redpencil.data.gift/id/deltas/producer/lblod-harvester-besluiten-producer",
+        "publisherUri": "http://data.lblod.info/services/delta-production-json-diff-file-manager-besluiten",
+        "queuePollInterval": 3000,
+        "relativeFilePath": "deltas/besluiten",
+        "serveDeltaFiles": "true",
+        "skipMuAuthInitialSync": "false",
+        "useVirtuosoForExpensiveSelects": "true"
+    },
+    "worship": {
+        "ACCOUNT": "http://lblod.data.gift/accounts/e98d2b13-a09c-4b56-968d-cbc11991f308",
```

Comment:
lower case & in delta-producer-publication-graph-maintainer, this seems to be an env variable and not part of the service config file